committee.rst

```diff
@@ -142,6 +143,11 @@ is not fixed.
 The nomination and voting process is kept private, by using direct
 email to committee members, not the mailing list. 
 
+Additionally, anyone may self-nominate themselves anytime and trigger a vote. 
```

Comment:
How, concretely, does a volunteer self-nominates out of season?

Reply:
Good point. By email to the secretary? (Public email would be bad.)

Reply:
An email to the secretary is probably the right way, indeed.. We just need to make it explicit.

Reply:
```suggestion
Additionally, anyone may self-nominate themselves anytime (by email to the secretary). 
```